common/script/fns/cron.js

```diff
@@ -36,6 +36,9 @@ module.exports = function(user, options) {
   }
   user.auth.timestamps.loggedin = new Date();
   user.lastCron = now;
+  if (_.isFinite(+user.preferences.timezoneOffset)) {
```

Comment:
Oh and it doesn't set user.preferences.timezoneOffsetAtLastCron to anything at all if user.preferences.timezoneOffset doesn't exist because the only time those two values should be different is if the user's timezone has changed.